### ISSUE TITLE
PI: network issues fetching binaries from `test-packetnet-ubuntu1604-x64-2`

### ISSUE THREAD

**Issue by refack** [nodejs/build#1479]
The fingerprint is:
```
ssh: connect to host 147.75.70.237 port 22: Connection timed out
```

This is highly intermittent, sometimes within the same job:
https://ci.nodejs.org/job/git-nodesource-update-reference/15200/console
(first call is `git remote-ls` succeeds, second call `git fetch` doesn't connect)

Ref: http://logs.libuv.org/node-build/2018-08-30

**Comment by refack** [comment]
P.S. since this is failing ~50% of [`node-test-commit-arm-fanned`](https://ci.nodejs.org/job/node-test-commit-arm-fanned/), I've disabled that job.

**Comment by refack** [comment]
@rvagg I had some idea for a possible long term solution.
1. set up a mini-VPN or a persistent (with some watchdog) ssh tunnel from "requireio" to `test-packetnet-ubuntu1604-x64-2`
2. get the cross compilers to push the binaries to "requireio" instead of to `test-packetnet-ubuntu1604-x64-2`
3. Piggyback on Jenkins's "artifacts" mechanism
4. use `scp` instead of `git` to propagate the binaries
5. use a dedicated storage server (e.g. rackspace block storage)

**Comment by richardlau** [comment]
Related to https://github.com/nodejs/build/issues/1483?

**Comment by refack** [comment]
> Related to #1483?

I don't think so, #1483 is WRT to ssh configuration to access from personal computer to requireio.
This issue is WRT to network problems from workers to the internet.
But maybe...

**Comment by refack** [comment]
It's back 😢 https://ci.nodejs.org/job/git-nodesource-update-reference/
```
08:38:00 debug1: Connecting to 147.75.70.237 [147.75.70.237] port 22.
08:38:10 debug1: connect to address 147.75.70.237 port 22: Connection timed out
08:38:10 ssh: connect to host 147.75.70.237 port 22: Connection timed out
```
https://ci.nodejs.org/job/node-test-commit-arm-fanned/ disabled for the time being.

**Comment by refack** [comment]
Resolved

**Comment by refack** [comment]
It's back - https://ci.nodejs.org/job/git-nodesource-update-reference/
